server/tools/docker-entrypoint.sh

```diff
@@ -223,12 +223,16 @@ echo ""
 echo "========================================================================="
 echo ""
 
-if [ "$DB_VENDOR" != "h2" ]; then
-    /bin/sh /opt/jboss/tools/databases/change-database.sh $DB_VENDOR
+configured_file="/opt/jboss/configured"
+if [ ! -e "$configured_file" ]; then
+    touch "$configured_file"
+    if [ "$DB_VENDOR" != "h2" ]; then
+        /bin/sh /opt/jboss/tools/databases/change-database.sh $DB_VENDOR
+    fi
+    /opt/jboss/tools/x509.sh
+    /opt/jboss/tools/jgroups.sh
```

Comment:
I believe we should grab all `*.sh` files into this `if` statement.

Reply:
Do you mean move the four other scripts below into this `if`? If you mean globbing, not all of `/opt/jboss/tools/*.sh` are run in this part of the script.

Reply:
Yes, I think all those scripts should be ran once.